src/adapters/scene.js

```diff
@@ -259,11 +260,27 @@ Scene.prototype.fitMap = function(item, padding) {
 }
 
 Scene.prototype.addMarker = async function(poi) {
+  const { className, subClassName, type } = poi
+  const icon = IconManager.get({ className, subClassName, type })
+
+  const element = document.createElement('div')
+
+  const div = document.createElement('div')
```

Comment:
The outer `div` is necessary as some styling is already applied by mapbox to position it at the right place? So if i wanted to reuse this `div` to add the content, it would interfere with the `transform`, the `right`, and `left` that i need to apply to the icon and the image.

I agree with with the naming `div`, i'm editing this right now 👍

Reply:
Perfect 👌